server/src/main/java/io/deephaven/server/table/stats/DateTimeChunkedStats.java

```diff
@@ -0,0 +1,61 @@
+package io.deephaven.server.table.stats;
+
+import io.deephaven.chunk.LongChunk;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.rowset.RowSet;
+import io.deephaven.engine.table.ChunkSource;
+import io.deephaven.engine.table.ColumnSource;
+import io.deephaven.engine.table.Table;
+import io.deephaven.engine.util.TableTools;
+import io.deephaven.time.DateTimeUtils;
+import io.deephaven.util.QueryConstants;
+
+public class DateTimeChunkedStats implements ChunkedStatsKernel {
```

Comment:
Seems to me like we could have just done these as objects.

Reply:
That could make sense, a few notes:
 * Collecting unique datetimes is _probably_ not constructive for many use cases
 * We would either lose min/max for datetime columns, or we could add min/max to character/object? It might be weird to have a min/max character, but maybe we don't care
 * We would need to stop reinterpreting to long (which I think means we would recreate each object on the way in), which I think is more expensive to compute?

Reply:
Yeah, I suppose you're right. Maybe we need comparables, and then we can have min-max *and* a sampling/count.